packages/select-rich/src/LionSelectRich.js

```diff
@@ -90,6 +89,13 @@ export class LionSelectRich extends ScopedElementsMixin(
 
       name: {
         type: String,
+        reflect: true,
```

Comment:
I think we already inherit this: https://github.com/ing-bank/lion/blob/master/packages/field/src/FormControlMixin.js#L37
We can omit name prop here

Reply:
Ah just checked, in old lion it was bugged because the form control mixin didn't have it. So I can remove it yea